cafebabel/static/js/textarea-change.js

```diff
@@ -1,8 +1,7 @@
 /* Auto-expand textarea when typing http://jsfiddle.net/hmelenok/WM6Gq/ */
 Array.from(document.querySelectorAll('textarea')).forEach(textArea => {
   function resize() {
-    textArea.style.height =
-      'auto' /* hack to allow textarea to be resized when deleting or cut content */
+    textArea.style.height = 0 /* textarea is NOT resized when deleting or cut content, to hack it : 'auto' (but it jumps when editing and saving) :-( */
```

Comment:
The comment is misleading, can you explain why there is `0`?

Reply:
I can't explain why it needed to be `auto` (since it's set instantly to `${textArea.scrollHeight}px`), nor I can explain why `0` fixes the jumping issue but prevent from resizing the textarea when deleting text. We can talk about it orally later, can't we?

Reply:
Any thoughts @davidbgk or @vinyll ?

Reply:
Proposal: `/* textarea is NOT resized when deleting or cutting content, to hack it, we tried 'auto' but it jumps when editing and saving so we had to fallback on '0' :-( */`